template/package.json

```
@@ -11,23 +11,23 @@
     "generate": "nuxt generate"
   },
   "dependencies": {
-    "nuxt": "^2.1.0",
-    "vuetify": "^1.2.6"
+    "nuxt": "^2.2.0",
+    "vuetify": "^1.3.2"
   },
   "devDependencies": {
     "babel-eslint": "^7.2.3",
```

```suggestion
     "babel-eslint": "^10.0.1",
```

Update to `"babel-eslint": "^10.0.1",`

---

@XanderLuciano Thank you updated